game/scripts/vscripts/TutorialGraph/Steps.ts

```diff
@@ -481,6 +481,33 @@ export const waitForModifierKey = (key: ModifierKey) => {
     })
 }
 
+/**
+ * Waits for the player to choose a chat wheel phrase.
+ * @param phraseIndex Optional chat wheel phrase index from 0 to 7 starting at the top going clockwise. If not passed will complete on any index.
```

Comment:
How are we supposed to know what line is on what index? Can we even know? I assume the lines will be localized to the player's client language.

Reply:
Right now the idea was just assuming that it's a new player that didn't change the lines.

If we really wanted to actually detect "Well played" we could localize the "Well played" string and then compare the text of each phrase to that.

Also if it isn't bound we can have a timeout on this section of 10 seconds or so.